versionology/has-version.go

```diff
@@ -0,0 +1,27 @@
+package versionology
+
+import "github.com/Masterminds/semver/v3"
+
+type HasVersion interface {
```

Comment:
I'd also love to see some comments for the `versionology` package. I don't think I fully get it.

Reply:
working on that now, will be pushed in the next commit